schema/library.raml

```diff
@@ -777,6 +776,12 @@ types:
       overlayVariable:
         type: VariableSpec
         required: false
+  CorrelationAssayMetadataBipartitenetworkPostRequest:
+    type: DataPluginRequestBase
+    properties:
+      computeConfig: EmptyDataPluginSpec
+      config: EmptyDataPluginSpec
+  CorrelationAssayMetadataStatsResponse: DifferentialAbundanceStatsResponse
```

Comment:
do we not know what the response will look like yet?

Reply:
we have a plan but since it doesn't matter for the frontend stub i didn't add it yet